infinitest-runner/src/main/java/org/infinitest/TestRunConfigurator.java

```diff
@@ -40,21 +40,20 @@
 	private static final Pattern EXCLUDED = Pattern.compile(PREFIX + EXCLUDED_GROUPS + SUFFIX);
 	private static final Pattern INCLUDED = Pattern.compile(PREFIX + INCLUDED_GROUPS + SUFFIX);
 	private static final Pattern LISTENER = Pattern.compile(PREFIX + LISTENERS + SUFFIX);
+	private static final Pattern EXCLUDED_CATEGORIES = Pattern.compile(PREFIX + "excluded-categories" + SUFFIX);
```

Comment:
I think we could reuse "groups" and "excluded-groups" for junit categories. That would simplify documentation and that's what maven surefire does in it's configuration so it's probably what wold be expected by users coming from maven.